src/equations/compressible_euler_2d.jl

```diff
@@ -1854,6 +1854,33 @@ end
     return SVector(w1, w2, w3, w4)
 end
 
+# Transformation from conservative variables u to entropy vector dSdu,
+# S = -rho*s/(gamma-1), s=ln(p)-gamma*ln(rho)
```

Comment:
That `S` is the mathematical entropy and not really the specific entropy that this function uses. I suggest the following change:
```suggestion
# Transformation from conservative variables u to entropy vector ds_0/du,
# using the specific entropy of Guermond et al. (2019): s_0 = p * rho^(-gamma) / (gamma-1).
# Note: This is *not* the "conventional" specific entropy s = ln(p / rho^(gamma)).
```
However, we can discuss if we want to add more information.

Reply:
I think I just copied the explanation of the wrong entropy from Fluxo when I added this routine. For me, it's good, as you suggested.

Reply:
Done in [ad7e1da](https://github.com/trixi-framework/Trixi.jl/pull/1792/commits/ad7e1dabf3c532d18b8af45b0b490b47b4a4e3ef)